site/content/in-dev/unreleased/getting-started/install-dependencies.md

```diff
@@ -0,0 +1,118 @@
+---
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+Title: Installing Dependencies
+type: docs
+weight: 100
```

Comment:
Where do these weights come from? I noticed it was 112 in the `deploying-polaris` subsections

Reply:
These are defined by us - they're what sorts the pages into whatever order we'd want (the larger the number, the further back it will be in the overall order). 112 is an odd number, let me look at that one again.

Update: I've changed those to 310/320/330 now in alignment with the overall section being a weight of 300 within the "Getting Started" section.